src/styles/blocks/name.scss

```diff
@@ -0,0 +1,16 @@
+.product__description {
```

Comment:
it's part of product block, so these styles should be inside of product.scss file. Don't create a separate file for that